_src/static/images/edx-svg/style.css

```diff
@@ -0,0 +1,14 @@
+.icon {
```

Comment:
Is this file created by Icomoon? If so, where is it referenced and used from (I couldn't find anywhere in this PR)?

In short, its a CSS file in the static/images/ directory. I'd prefer that we either 1) move it to our Sass/CSS directories as something we own or keep as a vendor file, but move it to vendor/.